hbase-common/src/main/java/org/apache/hadoop/hbase/io/ByteBuffAllocator.java

```diff
@@ -68,9 +68,25 @@
   // default heap allocator, it will just allocate ByteBuffers from heap but wrapped by an ByteBuff.
   public static final ByteBuffAllocator HEAP = ByteBuffAllocator.createOnHeap();
 
-  public static final String MAX_BUFFER_COUNT_KEY = "hbase.ipc.server.allocator.max.buffer.count";
```

Comment:
This config ``hbase.ipc.server.allocator.max.buffer.count`` and following ``hbase.ipc.server.allocator.buffer.size`` are introduced in HBASE-21879?

Reply:
Yeah, we have  wrapped all the allocate/deallocate op in an ByteBuffAllocator, was introduced in https://issues.apache.org/jira/browse/HBASE-21916.

Reply:
No wonder i can't find them, no more question.